Tasks/codepush-promote/task.json

```diff
@@ -103,15 +103,6 @@
             }
         },
         {
-            "name": "rollout",
```

Comment:
I would actually prefer to leave rollout as the top input for two reasons: it's the most common field you would set as part of a promote (the other ones are pretty advanced), and it doesn't inherit. Many of the other VSTS tasks sort fields by complexity not alphabetically, so I don't think it's matters that we sort the fields by their label.